services.php

```diff
@@ -99,12 +100,31 @@
      */
     'eddbk_render_cart_booking_info_handler' => function (ContainerInterface $c) {
         return new RenderCartBookingInfoHandler(
+            $c->get('eddbk_cart_booking_info_template'),
             $c->get('bookings_select_rm'),
             $c->get('sql_expression_builder'),
             $c->get('eddbk_cart/cart_items')
         );
     },
 
+    /**
+     * The template for booking info in the EDD cart.
+     *
+     * @since [*next-version*]
+     */
+    'eddbk_cart_booking_info_template' => function (ContainerInterface $c) {
```

Comment:
What do you think about using the factory to centralize some settings and make it easier to create individual templates?

Reply:
Good idea.